incubator/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -4240,7 +4194,6 @@ private void cleanupEnd(
                 long traceId,
                 long authorization)
             {
-                doSubscribeNetEnd(traceId, authorization);
```

Comment:
Now that `cleanupEnd` has reduced to `doSubscribeAppEnd`, we can replace calls to `cleanupEnd` with `doSubscribeAppEnd` and remove `cleanupEnd`.